This adds the option to disable nvenc, as for some reason it's not there.
It cannot be enabled on non-x86 platforms, yet the buildsystem forces it
for everything that is not freebsd and mac, which is wrong.

--- make/configure.py
+++ make/configure.py
@@ -1328,7 +1328,7 @@ def createCLI():
 
     h = IfHost( 'enable Nvidia NVEnc video encoder', '*-*-*', none=optparse.SUPPRESS_HELP ).value
     grp.add_option( '--enable-nvenc', dest="enable_nvenc", default=not (host.match( '*-*-darwin*' ) or host.match( '*-*-freebsd*' )), action='store_true', help=h )
-
+    grp.add_option( '--disable-nvenc', dest="enable_nvenc", action='store_false' )
 
     cli.add_option_group( grp )
 
